tap_quickbooks/__init__.py

```diff
@@ -26,7 +25,13 @@ def main():
         LOGGER.info("Starting discovery mode")
         catalog = do_discover()
         write_catalog(catalog)
+        LOGGER.info("Finished discovery mode")
```

Comment:
Can we add the same logger for sync mode?

Reply:
Added logger for finished sync mode.